hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/AllTypesInsertJdbcSqlConnectorTest.java

```diff
@@ -91,18 +92,11 @@ public static void beforeClass() {
 
     @Test
     public void insertRowWithAllTypes() throws Exception {
-        String tableName = randomTableName();
-        createTable(tableName, "id INT", "table_column " + type);
+        String tableName = generateTableName();
+        createTable_(tableName);
 
         String mappingName = "mapping_" + randomName();
-        execute("CREATE MAPPING " + mappingName
-                + " EXTERNAL NAME " + tableName
-                + " ("
-                + "id INT, "
-                + "table_column " + mappingType
-                + ") "
-                + "DATA CONNECTION " + TEST_DATABASE_REF
-        );
+        executeMapping(mappingName, tableName);
```

Comment:
I don't follow there changes.

Reply:
it was done before we had the conversation on createTable, once I implement createTable with quoted field names we will no longer need those changes.